Sources/OptionDescriptor.swift

```diff
@@ -1226,13 +1226,11 @@ struct _Descriptors {
         keyPath: \.preservedPrivateDeclarations
     )
 
-    let alphabetizeSwiftUIPropertyTypes = OptionDescriptor(
+    let swiftUIPropertiesSortMode = OptionDescriptor(
         argumentName: "sortswiftuiprops",
-        displayName: "Alphabetize SwiftUI Properties",
-        help: "Sorts SwiftUI properties alphabetically, defaults to \"false\"",
-        keyPath: \.alphabetizeSwiftUIPropertyTypes,
-        trueValues: ["enabled", "true"],
-        falseValues: ["disabled", "false"]
+        displayName: "Sort SwiftUI Dynamic Properties",
+        help: "Sorts SwiftUI dynamic properties, doesn't sort by default",
```

Comment:
We should include the list of possible arguments here, since this is the only documentation most users will see.

The 80 character limit can be a challenges on the option help text sometimes. Any of these seem reasonable to me:

```suggestion
        help: "Sort SwiftUI properties: none (default), alphabetize, or firstAppearanceSort",
```

```suggestion
        help: "Sort SwiftUI properties: none, alphabetize, or firstAppearanceSort",
```